packages/react/src/__tests__/ReactStrictMode-test.js

```diff
@@ -35,6 +35,9 @@ describe('ReactStrictMode', () => {
   });
 
   it('should appear in the client component stack', async () => {
+    // TODO REMOVE AFTER TESTING THIS FAILS ON PR
+    console.log('hit');
```

Comment:
I'm expecting the test to fail just to confirm the CI envar is set properly, then I'll remove this.